how-tos/working-locally/index.md

```diff
@@ -1,9 +1,11 @@
 ---
-title: Working with Cloud Data on your Local Machine
+title: How do I access Cloud Data from my Local Machine?
 ---
 
-The following How-to's are included in this section:
+The How-to guides in this section describe how to download data granules from Earthdata Cloud to your local machine.
 
-* Download from HTTPS
+_May be just a single how to guide_
```

Comment:
Maybe OPeNDAp could be another option here, unless we want to focus OPeNDAP under a subsetting header (which would be my suggestion).